ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -247,11 +254,14 @@ ADDTIME:                            'ADDTIME';
 DATE:                               'DATE';
 DATE_ADD:                           'DATE_ADD';
 DATE_FORMAT:                        'DATE_FORMAT';
+TIME_FORMAT:                        'TIME_FORMAT';
```

Comment:
a T spy in D party

Reply:
Good catch, updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/281/commits/6fa69170702814d04b302773f51e016cf64f7674